tests/unit/backend/wmg/api/test_expression_rollup.py

```diff
@@ -54,14 +56,326 @@ def test__cell_types_in_same_lineage_are_colinear(self):
+                ["UBERON:0000955", "CL:0002605", "multiethnic", 40],
+                ["UBERON:0000955", "CL:0002627", "HANCESTRO:0006", 10],
+                ["UBERON:0000955", "CL:0002627", "HANCESTRO:0008", 20],
+                ["UBERON:0000955", "CL:0002627", "multiethnic", 30],
+                ["UBERON:0000955", "CL:0002627", "unknown", 40],
+            ],
+            "expected_rolled_up_cell_counts": [
+                ["UBERON:0000955", "CL:0000127", "HANCESTRO:0005", 10],
+                ["UBERON:0000955", "CL:0000127", "HANCESTRO:0006", 10],
+                ["UBERON:0000955", "CL:0000127", "HANCESTRO:0008", 50],
+                ["UBERON:0000955", "CL:0000127", "multiethnic", 70],
+                ["UBERON:0000955", "CL:0000127", "unknown", 410],
+                ["UBERON:0000955", "CL:0000644", "unknown", 70],
+                ["UBERON:0000955", "CL:0002605", "HANCESTRO:0005", 10],
+                ["UBERON:0000955", "CL:0002605", "HANCESTRO:0008", 30],
+                ["UBERON:0000955", "CL:0002605", "multiethnic", 40],
+                ["UBERON:0000955", "CL:0002627", "HANCESTRO:0006", 10],
+                ["UBERON:0000955", "CL:0002627", "HANCESTRO:0008", 20],
+                ["UBERON:0000955", "CL:0002627", "multiethnic", 30],
+                ["UBERON:0000955", "CL:0002627", "unknown", 40],
+            ],
+            "input_gene_expression": [
+                ["ENSG00000085265", "UBERON:0000955", "CL:0000644", "unknown", 1, 1, 70, 1000],
+                ["ENSG00000085265", "UBERON:0000955", "CL:0002605", "HANCESTRO:0008", 1, 1, 30, 1000],
+                ["ENSG00000085265", "UBERON:0000955", "CL:0002605", "multiethnic", 1, 1, 40, 1000],
+                ["ENSG00000085265", "UBERON:0000955", "CL:0002627", "HANCESTRO:0008", 1, 1, 20, 1000],
+                ["ENSG00000169429", "UBERON:0000955", "CL:0002627", "multiethnic", 1, 1, 30, 1000],
+            ],
+            "expected_rolled_up_gene_expression": [
+                ["ENSG00000085265", "UBERON:0000955", "CL:0000127", "HANCESTRO:0008", 2, 2, 50, 1000],
+                ["ENSG00000085265", "UBERON:0000955", "CL:0000127", "multiethnic", 1, 1, 40, 1000],
+                ["ENSG00000169429", "UBERON:0000955", "CL:0000127", "multiethnic", 1, 1, 30, 1000],
+                ["ENSG00000085265", "UBERON:0000955", "CL:0000127", "unknown", 1, 1, 70, 1000],
+                ["ENSG00000085265", "UBERON:0000955", "CL:0000644", "unknown", 1, 1, 70, 1000],
+                ["ENSG00000085265", "UBERON:0000955", "CL:0002605", "HANCESTRO:0008", 1, 1, 30, 1000],
+                ["ENSG00000085265", "UBERON:0000955", "CL:0002605", "multiethnic", 1, 1, 40, 1000],
+                ["ENSG00000085265", "UBERON:0000955", "CL:0002627", "HANCESTRO:0008", 1, 1, 20, 1000],
+                ["ENSG00000169429", "UBERON:0000955", "CL:0002627", "multiethnic", 1, 1, 30, 1000],
+            ],
+        }
+
+        return [
+            (
+                test_1["name"],
```

Comment:
Are the test names being used anywhere (e.g. output in test logs)? If not, I'd move the names to comments above the different test cases.

Reply:
@atarashansky - yes, the names are used in the test output logs. For example if you run

```
pytest tests/unit/backend/wmg/api/test_expression_rollup.py -v
```

You will see test names in the output:

```
tests/unit/backend/wmg/api/test_expression_rollup.py::TestHighLevelRollupFunction::test__rollup_0_no_compare_dim_all_tissues_have_all_cell_types PASSED                                                                                       [ 57%]
tests/unit/backend/wmg/api/test_expression_rollup.py::TestHighLevelRollupFunction::test__rollup_1_no_compare_dim_one_ancestor_cell_type_missing_in_one_tissue_but_exists_in_all_others PASSED                                                 [ 71%]
tests/unit/backend/wmg/api/test_expression_rollup.py::TestHighLevelRollupFunction::test__rollup_2_with_ethnicity_compare_dim_on_single_tissue PASSED  
```